modules/dists/StencilDist.chpl

```diff
@@ -127,24 +127,33 @@ config param disableStencilLazyRAD = defaultDisableLazyRADOpt;
 //
 
 /*
-  The ``stencilDist`` distribution is a variant of the :mod:`blockDist
-  <BlockDist>` distribution that attempts to improve performance for
-  stencil computations by reducing the amount of communication
-  necessary during array accesses. From the user's perspective, it
-  behaves very similarly to ``blockDist`` in terms of reads, writes,
-  and iteration.
-
-  This distribution reduces communication by creating read-only caches for
-  elements adjacent to the block of elements owned by each locale. This
-  documentation may refer to these cached regions as 'ghost cells' or 'fluff'.
-  This approach can avoid many fine-grained GETs and PUTs when performing a
-  stencil computation near the boundary of the current locale's chunk of array
-  elements. The user must manually refresh these caches after writes by calling
-  the ``updateFluff`` method. Otherwise, reading and writing array elements
-  behaves the same as a block-distributed array.
-
-  The indices are partitioned in the same way as the :mod:`blockDist
-  <BlockDist>` distribution.
+  The ``stencilDist`` distribution is an enhanced functionality variant of the
+  :mod:`blockDist<BlockDist>` distribution. Both refer to an array partitioned
+  into blocks, but ``stencilDist`` will help the user to improve
+  performance for stencil computations by reducing the amount of communication
+  necessary for array accesses across locales. A ``stencilDist`` array would
+  most commonly be used in computations based on a numerical stencil formula.
+  For example, solving banded linear equations by the Gauss-Seidel or Jacobi
+  method.
+
+  Like ``blockDist``, each block in a ``stencilDist`` array is owned by a
+  (potentially) different locale. But when communication needs to occur between
+  a given block and its set of immediately neighboring or adjacent blocks, the
+  more feature-full ``stencilDist`` most likely makes it the better choice.
```

Comment:
"full" should be "ful" here, and as long as we're changing the line, I'd say "more feature-rich ``stencilDist`` is typically the better choice."